coverage.sh

```diff
@@ -3,7 +3,7 @@
 #
 # MIT License
 #
-# Copyright (c) 2017-2018 Thibault Martinez and Simon Ninon
+# Copyright (c) 2017-2018 Charles Fraïssé and Michaël Leclerc
```

Comment:
MIT requires giving credit to author, this this file is copied from their repositories, we have to use their header